src/observe.ts

```diff
@@ -18,8 +18,12 @@ export const initPerformanceObserver = (): void => {
   onTTFB(report => {
     // Calculate the request time by subtracting from TTFB
     // everything that happened prior to the request starting.
+    //
+    // report.entries[0] will be missing when page was restored from Back-Forward Cache.
+    // In this case, `0` value will be reported, and perfume.js users can decide how to handle it (report `0` or ignore it)
+    //
     // @ts-ignore
-    report.value = report.value - report.entries[0].requestStart;
+    report.value = report.entries[0] ? report.value - report.entries[0].requestStart : report.value;
```

Comment:
I feel like we should just take a step forward and not report it completely.

Otherwise, it will increase the score for "good" pages. And I have a feeling people will not read the docs, and forget to skip the `0` value.

Reply:
Actually never mind, let me read this convo first. https://github.com/GoogleChrome/web-vitals/issues/313

Reply:
@SuperOleg39 I read the discussion, which btw, thank you @tunetheweb for taking the time to respond with so many insights. 

I was not aware of this definition "[CrUX measured "TTFB"](https://stackoverflow.com/questions/69116839/does-ttfb-include-the-time-for-the-request-that-redirects-to-my-page)", that's brilliant.

![ttfb](https://user-images.githubusercontent.com/829214/216746892-db553922-9652-4dc7-8c81-e18139c0168e.png)


As Perfume.js aims to be used for capturing data similar to "CrUX", let's follow that standard.

@SuperOleg39 would you mind taking care of adjusting TTFB to "CrUX" standard, adding the tests and updating the README please.

As this is a breaking changes, this means we are going to cut v9 with this fix.

Reply:
Updated onTTFB handle, will try to add some tests later, need to think about realization)